simul/simul.go

```diff
@@ -260,3 +304,32 @@ func getStartStop(rcs int) (int, int) {
 	dbg.Lvl2("Range is", start, ":", stop)
 	return start, stop
 }
+
+// getRunWait returns either the command-line value or the value from the runconfig
+// file
+func getRunWait(rc platform.RunConfig) int {
```

Comment:
? You just pass the default values into the `GetInt` function ? 
You go from `getRunWait(rc)` to `rc.GetInt("runwait", runwait)` 
Writing an issue for it.